store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/AccumuloStoreTest.java

```diff
@@ -102,21 +109,41 @@
 import static uk.gov.gchq.gaffer.store.StoreTrait.VISIBILITY;
 
 public class AccumuloStoreTest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AccumuloStoreTest.class);
     private static final String BYTE_ENTITY_GRAPH = "byteEntityGraph";
     private static final String GAFFER_1_GRAPH = "gaffer1Graph";
-    private static SingleUseMockAccumuloStore byteEntityStore;
-    private static SingleUseMockAccumuloStore gaffer1KeyStore;
+    private static AccumuloStore byteEntityStore;
+    private static AccumuloStore gaffer1KeyStore;
     private static final Schema SCHEMA = Schema.fromJson(StreamUtil.schemas(AccumuloStoreTest.class));
     private static final AccumuloProperties PROPERTIES = AccumuloProperties.loadStoreProperties(StreamUtil.storeProps(AccumuloStoreTest.class));
     private static final AccumuloProperties CLASSIC_PROPERTIES = AccumuloProperties.loadStoreProperties(StreamUtil.openStream(AccumuloStoreTest.class, "/accumuloStoreClassicKeys.properties"));
+    private static AccumuloTestClusterManager accumuloTestClusterManagerByteEntity = null;
+    private static AccumuloTestClusterManager accumuloTestClusterManagerGaffer1Key = null;
+
+    @ClassRule
+    public static TemporaryFolder storeBaseFolder = new TemporaryFolder(CommonTestConstants.TMP_DIRECTORY);
 
     @Rule
     public ExpectedException thrown = ExpectedException.none();
 
     @BeforeClass
     public static void setup() {
-        byteEntityStore = new SingleUseMockAccumuloStore();
-        gaffer1KeyStore = new SingleUseMockAccumuloStore();
+        File storeFolder1 = null;
```

Comment:
I have found that sometimes multiple TemporaryFolder entries get set by JUnit4 with the same file store location.
Additionally, TemporaryFolder is deprecated in the coming JUnit5 changes so this will be quite short lived anyway.
I will leave as is, if that is OK with you.